docs/scripts/check-closing-quotes.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+# Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+whitelist="$0.whitelist"
+
+echo Checking closing quotes '`' for all rst file.
+
+errors=0
+
+while read file
+do
+  numberOfQuotes=$(egrep --invert-match --file="$whitelist" "$file" | grep --only-matching '`' | wc --lines)
+  if (( $numberOfQuotes % 2 ))
+  then
+    ((errors++))
+    echo "$file" : $numberOfQuotes
+    echo Suspicious lines:
+    awk -F '`' 'NF % 2 == 0 && NF > 0 ' < "$file"
```

Comment:
```suggestion
    grep --perl-regexp '^(?!([^`]*(((`[^`]*`)|(``[^`]*``))[^`]*)*)$)' < "$file" 
```
Maybe like this?

Reply:
`grep` and `awk` in particular come in a wide variety of subtly-different option sets, so please do test that under dev-env before final commit. If you do end up preferring `awk` over `grep`, bear in mind that the one in dev-env is called `gawk`.

Reply:
thanks! I tested both `gawk` and `grep` , they both work the same way under dev-env.